test/recipes/99-test_fuzz_dtlsclient.t

```diff
@@ -16,7 +16,7 @@ my $fuzzer = "dtlsclient";
 setup("test_fuzz_${fuzzer}");
 
 plan skip_all => "This test requires dtls support"
-    if disabled("dtls");
+    if disabled("sock");
```

Comment:
I think this is wrong and accounts for some of the CI failures: you can disable dtls without disabling sock and then this fuzzer shouldn't run.

Reply:
yes, I confused myself.